tests/conftest.py

```diff
@@ -41,6 +41,27 @@ def ctx_model():
     plugin = cam.genericls.Plugin()
     return plugin.from_isd(csm_isd, plugin.modelname(1))
 
+@pytest.fixture
+def hrsc_nadir_model():
+    path = os.path.join(data_path, 'h0232_0000_nd2_keywords.lis')
+    csm_isd = isd.Isd.read_socet_file(path)
+    plugin = cam.genericls.Plugin()
+    return plugin.from_isd(csm_isd, plugin.modelname(1))
+
+@pytest.fixture
+def hrsc_stereo_1_model():
+    path = os.path.join(data_path, 'h0232_0000_s12_keywords.lis')
+    csm_isd = isd.Isd.read_socet_file(path)
+    plugin = cam.genericls.Plugin()
+    return plugin.from_isd(csm_isd, plugin.modelname(1))
+
+@pytest.fixture
+def hrsc_stereo_2_model():
+    path = os.path.join(data_path, 'h0232_0000_s22_keywords.lis')
+    csm_isd = isd.Isd.read_socet_file(path)
+    plugin = cam.genericls.Plugin()
+    return plugin.from_isd(csm_isd, plugin.modelname(1))
+
```

Comment:
are you thinking that these fixtures will be widely used or are they tightly coupled to the classes with like names?  If the latter, perhaps move them inside the class since no other test is likely to need these fixtures?